scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -296,21 +324,24 @@ class FossId(
                 checkNotNull(scans)
 
                 val existingScan = scans.sortedByDescending { it.id }.find { scan ->
-                    scan.gitBranch == revision && scan.gitRepoUrl == url
+                    // The scans in the server contain the url with the credentials so we have to remove it for the
```

Comment:
This change does not seem to be related to the naming conventions, or is it?

Reply:
Moved to another commit.